### PR TITLE
Bugfix/button does not trigger

### PR DESCRIPTION
**Pull request by LucaMele**
Fixes some issues after with-react merge

### CODE REVIEW

File: src/components/10-atoms/button/index.js [hunk 1]
```diff
@@ -81,11 +81,11 @@ class AXAButton extends LitElement {
     // If someone fires a click on the button and its type is submit then trigger fake button
     // press
     this.onclick = e => {
-      this.handleClick(e);
+      this.handleClick(e, true);
     };
   }
 
-  handleClick = e => {
+  handleClick = (e, ignoreOnClick = false) => {
```
Comment (danyball) [inline src/components/10-atoms/button/index.js]: I was a bit confused about that attribute `ignoreOnClick`. A function "handleClick" with an attribute "ignoreOnClick" is worthless?! Perhaps a more accurate name would be better to read.

File: src/components/10-atoms/button/index.js [hunk 2]
```diff
@@ -95,6 +95,10 @@ class AXAButton extends LitElement {
       e.stopPropagation();
       this.fakeButton.click();
     }
+
+    if (!ignoreOnClick && typeof this.onClick === 'function') {
+      this.onClick(e);
```
Comment (danyball) [inline src/components/10-atoms/button/index.js]: You are calling `onClick` - is this the same method like in line 83 `onclick`? If so, a endless circuit could happen: Line 84 calls handleclick -> handleclick calls onclick on line 100 -> .......

**Review by danyball — CHANGES_REQUESTED** [review]

File: src/components/10-atoms/button/index.js (hunk 1 above)
Comment (LucaMele) [reply src/components/10-atoms/button/index.js]: @danyball unfortunately is necessary. the comment:    `// If someone fires a click on the button and its type is submit then trigger fake button` should explain why

**Review by danyball — APPROVED** [review]
I run that code with storybook > react example. button works. I dont know the bug, but your result works. So its fine for me ;)

**Comment by LucaMele** [comment]
thank you very much

**Pull request merged**